mreg/api/permissions.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+from rest_framework.permissions import BasePermission
+
+class IsInRequiredGroup(BasePermission):
+    """
+    Allows only access to users in the required group.
+    """
+
+    def has_permission(self, request, view):
+        if not bool(request.user and request.user.is_authenticated):
+            return False
+        REQUIRED_USER_GROUP = getattr(settings, 'REQUIRED_USER_GROUP', None)
+        if REQUIRED_USER_GROUP is None:
+            return True
```

Comment:
Sane to `return True` here? Maybe raise an error.

Reply:
I'd raise an error